src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.body {
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.img {
+  position: absolute;
+}
+
+.img-lg {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.img-sm {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-lg {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.search-sm {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 12px;
+  line-height: 19px;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+}
+
+.search-lg::placeholder {
+  font-size: 16px;
+}
+
+.search-sm::placeholder {
+  font-size: 14px;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```

Comment:
This border was already set in `search` class, so there is no need to duplicate this value